easybuild/easyconfigs/g/GNU/GNU-4.9.3-2.25.eb

```diff
@@ -12,9 +12,9 @@ toolchain = {'name': 'dummy', 'version': 'dummy'}
 
 dependencies = [
     # GCC built on top of (dummy-built) binutils
-    ('GCC', gccver, '-binutils-%s' % binutilsver),
```

Comment:
I'm trying to make this work exactly like it did before, with the renaming that this PR does this means GCC -> GCCcore